src/styles/modules/_m-paragraph.sass

```diff
@@ -21,8 +21,8 @@ $sizes: ( xxxs: $font-size-xxxs, xxs: $font-size-xxs, xs: $font-size-xs, s: $fon
   +m('color-blue')
     color: color(ablue)
 
-  +m('family-museo')
-    font-family: $font-museo
+  +m('family-special')
+    font-family: $font
```

Comment:
special should be 'Bayu Prahara, Bradley Hand ITC, cursive' not the same as $font.  It should be $font-special